deployments/azure-functions-app-container.yml

```diff
@@ -0,0 +1,57 @@
+# Action Requires
+# 1. Setup the AZURE_CREDENTIALS secrets in your GitHub Repository
+# 2. Setup the REGISTRY_USERNAME secrets in your GitHub Repository
+# 3. Setup the REGISTRY_PASSWORD secrets in your GitHub Repository
+# 4. Replace REGISTRY, NAMESPACE, IMAGE, TAG in the following template with proper values
+# 5. Add this yaml file to your project's .github/workflows/
+# 6. Push your local project to your GitHub Repository
+
+name: Linux_Container_Workflow
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    environment: dev
+    steps:
+    - name: 'Checkout GitHub Action'
+      uses: actions/checkout@v3
+
+    - name: 'Login via Azure CLI'
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: 'Docker Login'
+      uses: azure/docker-login@v1
+      with:
+        login-server: contoso.azurecr.io
+        username: ${{ secrets.REGISTRY_USERNAME }}
+        password: ${{ secrets.REGISTRY_PASSWORD }}
+
+    - name: 'Compose Customized Docker Image'
+      shell: bash
+      run: |
+        # If your function app project is not located in your repository's root
+        # Please change the path to your directory for docker build
+        docker build . -t REGISTRY/NAMESPACE/IMAGE:TAG
```

Comment:
we should use environment variables for REGISTRY, NAMESPACE, IMAGE and TAG values, so that user have to change at only one place.